task-4/questions-project/src/App.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from "react";
+
+import FinalPage from "./FinalPage";
+import { QUESTIONS } from "./utils/constant";
+import {
+  QuestionHeader,
+  MainContainer,
+  Container,
+  Answer,
+  AnswerText,
+  QuestionImage,
+} from "./App.styled";
+
+const CORRECT_ANSWER_MESSAGE = "Вы ответили верно!";
+
+function App() {
```

Comment:
Let's rewrite the component

Will have only this useState

```
const [questionIndex, setQuestionIndex] = useState(0);
  const [clickedAnswer, setClickedAnswer] = useState<string | null>(null);
  const [isFinalPage, setIsFinalPage] = useState(false);
```

Reply:
and this function - `onAnswerClick`

Reply:
For other - color & text - you do not need store it in useState, you can calculate this via clickedAnswer & questionIndex